### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by jedgarpark**
Not sure if the ```global char_count, current_mode``` line is a problem in these?

### CODE REVIEW

**Comment by caternuson** [comment]
I'm OK with just using global and telling pylint to ignore.

Did you run this through black? Curious what spread out the line formatting so much.

**Comment by jedgarpark** [comment]
Yes, that was from running it through Black @caternuson

File: Baudot_TTY/baudot_tty_ble.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    baudot_start()  # send start bit tone
+    for i in range(5):  # send each bit of the character
+        bit = (value >> i) & 0x01  # bit shift and bit mask to get value of each bit
+        baudot_bit(tones[bit])  # send each bit, either 0 or 1, of a character
+    baudot_stop()  # send stop bit
+    baudot_carrier()  # not to spec, but works better to extend carrier
+
+
+def send_message(text):
+    global char_count, current_mode  # pylint: disable=global-statement
```
Comment (jepler) [inline Baudot_TTY/baudot_tty_ble.py:176]: I think this pylint disable is fine.  You _could_ re-organize this as a class with properties but it's not likely to be worth the effort.

Comment (jedgarpark) [reply Baudot_TTY/baudot_tty_ble.py:176]: If you're OK I'm OK.

**Pull request merged**